.gitlab-ci.yml

```diff
@@ -25,6 +25,7 @@ dcp_wide_test_SS2:
   only:
     - integration
     - staging
+    - validate-schema-versions
```

Comment:
I think I'm a little confused why `validate-schema-versions` is an environment listed here and likewise below?